src/components/atoms/ButtonGroupPicker/stories/ButtonGroupPicker.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { Button } from '../../../atoms/Button/Button'
+import { ButtonGroupPicker } from '../ButtonGroupPicker'
+import Wrapper from '../../../_helpers/Wrapper'
+
+export default {
+  title: '💧 Atoms/ButtonGroupPicker',
+  component: ButtonGroupPicker,
+}
+
+export const Basic = () => {
+  const [selectedValue, setSelectedValue] = React.useState(1)
+
+  return (
+    <Wrapper>
+      <ButtonGroupPicker>
+        {[1, 2, 3].map(value => {
```

Comment:
maybe it would be nice to have a prop for the groupPicker to set a number of buttons instead of only use children (or both options), but for now this is okay ^^